setup.py

```diff
@@ -29,7 +29,7 @@
     print("Only Python 2.7 is supported.")
     sys.exit(1)
 
-import commands
+import subprocess
```

Comment:
subprocess came in 2.4 (https://docs.python.org/2/library/subprocess.html#module-subprocess), so not sure how backwards compatible this change would be. Not following pysqlite history, so no idea if this is something to worry about.